WalletWasabi/JsonConverters/GuidJsonConverter.cs

```diff
@@ -9,18 +9,6 @@ namespace WalletWasabi.JsonConverters
 	/// </summary>
 	public class GuidJsonConverter : JsonConverter
 	{
-		/// <summary>
-		///     Gets the instance.
-		/// </summary>
-		public static GuidJsonConverter Instance { get; } = new GuidJsonConverter();
-
-		/// <summary>
-		///     Gets a value indicating whether this <see cref="T:Newtonsoft.Json.JsonConverter"/> can read JSON.
-		/// </summary>
-		/// <value><see langword="true"/> if this <see cref="T:Newtonsoft.Json.JsonConverter"/> can read JSON; otherwise, <see langword="false"/>.
-		/// </value>
-		public override bool CanRead => true;
```

Comment:
This overrides `CanRead` parent property. This requires careful explanation why it is safe.

Reply:
It is okay to remove. It overrides the property to set the same value that its parent so, it is redundant and unnecessary.